### PR TITLE
header - datadirization

### PR DESCRIPTION
**Pull request by pmauduit**
Nothing new, same methods as the other datadirizations.

* [x] build
* [x] runtime tests with `georchestra.datadir`
* [x] runtime tests without `georchestra.datadir`

### CODE REVIEW

File: header/pom.xml [hunk 1]
```diff
@@ -129,4 +140,110 @@
...
+          <plugin>
+            <groupId>net.sf.debian-maven</groupId>
+            <artifactId>debian-maven-plugin</artifactId>
+            <version>1.0.6</version>
+            <configuration>
+              <packageName>georchestra-header</packageName>
+              <packageDescription>Debian package for the geOrchestra's Security Proxy</packageDescription>
+              <projectOrganization>geOrchestra</projectOrganization>
+              <maintainerName>Pierre Mauduit</maintainerName>
+              <maintainerEmail>pierre.mauduit@gmail.com</maintainerEmail>
```
Comment (fvanderbiest) [inline header/pom.xml]: Use psc@georchestra.org instead, thanks :-)

File: header/src/main/filtered-resources/WEB-INF/jsp/index.jsp [hunk 2]
```diff
@@ -215,7 +240,7 @@ if(sec_roles != null) {
         <c:choose>
             <c:when test='<%= anonymous == false %>'>
         <p class="logged">
-            <a href="${ldapadminPublicContextPath}/account/userdetails"><%=request.getHeader("sec-username") %></a><span class="light"> | </span><a href="/j_spring_security_logout"><fmt:message key="logout"/></a>
+            <a href="<%=ldapadm %>/account/userdetails"><%=request.getHeader("sec-username") %></a><span class="light"> | </span><a href="/j_spring_security_logout"><fmt:message key="logout"/></a>
```
Comment (fvanderbiest) [inline header/src/main/filtered-resources/WEB-INF/jsp/index.jsp]: The double slash (conveyed by ```ldapadm```) in the path is incorrect here

File: header/src/deb/resources/etc/georchestra/header/header.properties [hunk 3]
```diff
@@ -0,0 +1,2 @@
+language=fr
+ldapadminPublicContextPath=//ldapadmin/
```
Comment (fvanderbiest) [inline header/src/deb/resources/etc/georchestra/header/header.properties]: ldapadminPublicContextPath=/ldapadmin/

**Comment by fvanderbiest** [comment]
2 issues to fix and it's good to merge :-)

File: header/src/main/filtered-resources/WEB-INF/jsp/index.jsp (hunk 2 above)
Comment (pmauduit) [inline header/src/main/filtered-resources/WEB-INF/jsp/index.jsp]: If fixed into the .properties file, there is nothing more to do here, right ? (or else I did not get the comment)

Comment (fvanderbiest) [inline header/src/main/filtered-resources/WEB-INF/jsp/index.jsp]: right

**Pull request merged**